Edit daemon commands for Linux (#875)
zkSNACKs/WasabiDoc: docs/using-wasabi/Daemon.md

### Linux

If the package is installed, open the command line terminal and execute in any directory:

```bash
wassabee mix --wallet:MyFirstWallet --destination:MySecondWallet --keepalive
```

> If the source code is built, open the command line terminal and execute:

Reviewer: ```suggestion
If the source code is built, open the terminal, navigate to the `WalletWasabi.Gui` folder inside the cloned repository, and execute:
```

```bash

> cd ~/WalletWasabi/WalletWasabi.Gui

Reviewer: ```suggestion
```

If the package is installed, execute in the command line in any directory: 

```bash
wassabee mix --wallet:MyFirstWallet --destination:MySecondWallet --keepalive
```

> If the source code is built, open the Command Line and navigate to the WalletWasabi.Gui folder (inside the cloned repo) and execute:

Reviewer: ```suggestion
If the source code is built, open the command line terminal and execute:
```

```bash

> dotnet run -- mix --wallet:MyFirstWallet --destination:MySecondWallet --keepalive

Reviewer: ```suggestion
cd ~/WalletWasabi/WalletWasabi.Gui
dotnet run -- mix --wallet:MyFirstWallet --destination:MySecondWallet --keepalive
```

Author: If you open the Terminal anywhere and you execute the first command, will it the directory change to `WalletWasabi.Gui`  folder no matter where it is?

If yes then what happens when you have two repos? https://github.com/zkSNACKs/WasabiDoc/pull/874#discussion_r432231047

Reviewer: Yes. the `~/` denotes the root directory. Thus, if the user did `~$ git clone github.com/zksnacks/WalletWasabi`, as is earlier in the guide, then the Gui folder will be at that path.

Sure, if there are two repos `WalletWasabi1` and `WalletWasabi2`, then this would not work. But, this is only a small sub section of users who will clone the repo twice, as it is not mentioned in the docs to do so. And whoever clones the twice probably knows how to change directories anyhow...

Author: > Yes. the ~/ denotes the root directory. Thus, if the user did ~$ git clone github.com/zksnacks/WalletWasabi, as is earlier in the guide, then the Gui folder will be at that path.

> Sure, if there are two repos WalletWasabi1 and WalletWasabi2, then this would not work.

Not only that, even if you have only one Wasabi repo but it is not in the root directory then it will not work.

My point is using `cd ~/WalletWasabi/WalletWasabi.Gui` will only work assuming that the repo is somewhere, while saying `open the Command Line and navigate to the WalletWasabi.Gui folder (inside the cloned repo)` doesn't assume anything.
And if the user is not able to navigate to a folder or is not willing to learn how to do that, how do you expect them to clone the repo and use the daemon?

Reviewer: That is a good point. 
But, if the user does a step by step according to the docs, the GUI folder will be on this path specifically. Sure, when he deviates from the guide, he cannot expect that it still works.

I'm kinda torn here... I know several whom I helped to build source, and they struggle with basic commands like `cd` already... Thus, I'd like this guide to be fool proof. I'd prefer it to show every [reasonable] action that the user has to take, and this included `cd`.

> If the package is installed, execute in the command line in any directory: 

Reviewer: ```suggestion
If the package is installed, open the command line terminal and execute in any directory:
```
